zerver/tests/test_bots.py

```diff
@@ -1501,7 +1501,10 @@ def test_patch_bot_via_post(self) -> None:
         email = "hambot-bot@zulip.testserver"
         # Important: We intentionally use the wrong method, post, here.
         result = self.client_post(f"/json/bots/{self.get_bot_user(email).id}", bot_info)
-        response_dict = self.assert_json_success(result)
+
+        # TODO: The "method" parameter is not currently tracked as a processed parameter
+        # by has_request_variables. Assert it is returned as an ignored parameter.
+        response_dict = self.assert_json_success(result, ["method"])
```

Comment:
We likely just need to have `zerver.lib.rest.get_target_view_function_or_response` adding `method` to processed parameters.

Reply:
True. I'm thinking it would be an undocumented parameter, similar to how we process `client` in `zerver.middleware.parse_client`?

Reply:
Yeah, it would be quite similar.